greeting.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+
+namespace Greeting
+{
+    public class greeting
+    {
+        public static void Main(string[] args)
+        {
+            Console.Write("Hello! What is your name?");
+            string name = Console.ReadLine();
+            string time = Convert.ToString(DateTime.Now);
+            int mins = Convert.ToInt32(time.Substring(14, 2));
+            int hours = Convert.ToInt32(time.Substring(11, 2)) + 3; //+3 because of belarus timezone
```

Comment:
instead of this, you can use the following approach to get hours as a number:
`int hours = DateTime.Now.Hour;`

Reply:
and you do not need to use minutes